card/mod/core/set/all/states.rb

```diff
@@ -1,13 +1,12 @@
 module ClassMethods
-  def exists? mark
-    card = quick_fetch mark
-    card.present?
+  def real? mark
```

Comment:
I think we should deprecate `exists?` and only use `real?`.  More consistent.

Reply:
but more inconsistent with Rails

Reply:
ok, maybe we don't deprecate, but we do make `real?` our best practice? 

Alternatively, we could go the other way and deprecate `:real` as a status in favor of `:exists`